_targets.R

```diff
@@ -18,7 +19,18 @@ list(
   tar_target(oldest_active_sites, find_oldest_sites(states, parameter)),
 
   # TODO: PULL SITE DATA HERE
-
+  tar_target(
+    wi_data,
+    get_site_data(oldest_active_sites, 'WI', '00060')
```

Comment:
You already have the `states` and `parameter` defined in the objects above. I would suggest switching these to use `states[1]`, `states[2]`, and `states[3]` in place of the state abbreviation strings that you have & replace all of the `'00060'` strings in your `get_site_data()` calls with `parameter`